Makefile

```diff
@@ -70,13 +70,13 @@ dev-agent-image:
 	$(eval arch := $(shell uname -m))
 	CGO_ENABLED=0 GOOS=linux go build -o build/zarf-linux-$(arch) main.go
 	DOCKER_BUILDKIT=1 docker build --tag $(tag) --build-arg TARGETARCH=$(arch) . && \
-	kind load docker-image zarf-agent:$(tag) && \
+	kind load docker-image $(tag) && \
 	kubectl -n zarf set image deployment/agent-hook server=$(tag)
 
 init-package: ## Create the zarf init package, macos "brew install coreutils" first
 	@test -s $(ZARF_BIN) || $(MAKE) build-cli
 
-	@test -s ./build/zarf-init-$(ARCH).tar.zst || $(ZARF_BIN) package create -o build -a $(ARCH) --confirm .
```

Comment:
I dont hate this change, except this will force local tests to always rebuild the init package--which often isn't necessary during dev, is there a particular reason to keep the behavior consistent with the examples?

Reply:
`make e2e` builds the example packages as a side affect.
`make init-package` builds the init-package as its primary purpose. 

When I was testing changes with the zarf-agent it took me longer than I'm proud to admit to realize that I wasn't actually rebuilding the init-package 😛

Reply:
Yeah that's fair, then can we make a new `make init-package-if-not-exist` or something that e2e calls--and we could do the same for examples too I suppose, but that will make things noisy in the makefile.  Alternatively, we could just dump a warning in the makefile output, that the file already exists so reusing it, this is just for devs after all.  You can change text colors in make pretty easy:

```shell
GREEN  := $(shell tput -Txterm setaf 2)
YELLOW := $(shell tput -Txterm setaf 3)
WHITE  := $(shell tput -Txterm setaf 7)
CYAN   := $(shell tput -Txterm setaf 6)
RESET  := $(shell tput -Txterm sgr0)
```

```shell
 ${YELLOW}words in yellow${RESET} ${GREEN}words in green${RESET}
```

Reply:
How about a middle ground where we keep the `init-package` explicit and always creates the package but then I update the workflows.yaml and the `test-e2e` target to only make the init-package if it's not already there?

Reply:
that seems reasonable to me